dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_1_general.rst

```diff
@@ -1,71 +1,94 @@
-- After data is loaded, the *Grouping Options* and *Data Analysis* tabs should be enabled.
+**Time required 5 minutes**
+
+- Open **Muon Analysis** (*Interfaces* > *Muon* > *Muon Analysis*)
+- On the **Home** tab, in the *Instrument* section, set the instrument to 
+  **EMU**
+- Load run ``20918``
 - The data should look like this:
 
 .. image:: /images/MuonAnalysisTests/emu20918.png
   :align: center
+  :alt: emu20918.png
 
-(settings used on *Settings* tab are: finish 10 us, autoscale on, line plot, no rebinning).
-
-- Go to *Settings* tab and, under "Rebin data", select "Fixed" with steps of 5. Should now look like this:
+- On the **Home** tab, under the *Rebin* section, change **Rebin** to 
+  **Fixed** with steps of ``5``. The data should now look like this:
 
 .. image:: /images/MuonAnalysisTests/emu20918_rebin.png
   :align: center
+  :alt: emu20918_rebin.png
 
-Test 3: Old style data analysis GUI
+-----------------------------------
+  
+Test 3: Old Style Data Analysis GUI
 -----------------------------------
 
-- Still on the *Settings* tab from test 2, leave rebinning on and make sure that "Enable multiple fitting" is turned **off**. (It should be off by default).
-- Go to the *Data Analysis* tab. The interface should contain a fit property browser. The ``TF Asymmetry Mode`` should be unticked. 
-- Set up an ``Abragam`` function with ``Omega=8.5``, ``Tau=0.5`` and fit the data. The fit should be done to the rebinned data.
-
-Test 4: Fit to raw data
------------------------
-
-- Now tick the ``Fit to raw data`` box in the bottom section and fit again. This time it will fit to the raw data, but the rebinned data will still be shown in the plot.
+**Time required 5-10 minutes**
 
-Test 5: New style data analysis GUI
------------------------------------
+- Start with the same data loaded and rebin settings from :ref:`test_2`
+- Go to the **Fitting** tab
+	- Right click the empty table area; Select **Add Function**
+	- Add **Abragam** (*Muon* > *MuonSpecific* > *Abragam*)
+	- Set ``Omega = 8.5`` and ``Tau = 0.5``
+	- In the bottom table, check **Fit To Raw Data** to be ``false``
+	- Click **Fit**
+- The fit should be applied to the rebinned data
 
-- On the *Settings* tab, turn "Enable multiple fitting" **on**.
-- Go back to *Data Analysis*. The new UI should be there, and the fit function should have been cleared (so the Fit option is disabled until a new function is set up).
-- Set up the Abragam function as in test 3 and fit.
-- Try this with and without "Fit to raw data" as in test 4.
+.. image:: /images/MuonAnalysisTests/emu20918_rebin_fitted.png
+	:align: center
+	:alt: emu20918_rebin_fitted.png
 
-Teardown
---------
+- Now set **Fit To Raw Data** to be ``true`` and click **Fit** again. This time 
+  it will fit to the raw data, but the rebinned data will still be shown in the 
+  plot
+- There should now be two new workspaces in the workspace toolbox; One for 
+  fitted raw data and one for fitted rebinned data
```

Comment:
```suggestion
  fitted raw data and one for fitted rebinned data.
```

Reply:
What's the point of a full stop here?

Reply:
end of a sentence.

Reply:
I haven't used a full stop for any of the other testing instructions

Reply:
ok, never mind then